docker-compose-rule-events-api/src/main/conjure/report-api.yml

```diff
@@ -0,0 +1,45 @@
+types:
+  conjure-imports:
+    events: event-api.yml
+
+  definitions:
+    default-package: com.palantir.docker.compose.report
+    objects:
+      GitInfo:
+        fields:
+          branch: optional<string>
+          commit: optional<string>
+          dirty: optional<boolean>
+
+      TestDescription:
+        fields:
+          testClass: optional<string>
+          displayName: optional<string>
+          method: optional<string>
+
+      DockerComposeRun:
+        fields:
+          runId: string
+          startTime: datetime
+          finishTime: datetime
+          events: list<events.Event>
+          dockerComposeConfig: optional<string>
+          testDescription: TestDescription
+          exceptions: list<string>
```

Comment:
I wonder if it would be useful to grab the actual @Test success/failure info?

Reply:
It would, but we can't as sometimes DCR is run a @ClassRule which doesn't have access to individual tests